_includes/pbc/all/install-features/202505.0/install-the-order-amendment-feature.md

```diff
@@ -546,97 +554,107 @@ sales_order_amendment_oms.validation.order_not_being_amended,This order cannot b
+| ItemMetadataSalesOrderItemCollectionPostUpdatePlugin              | Processes item metadata after order items collection update.                                                                                                                                                | None                                                                                                                                             | Spryker\Zed\SalesProductConnector\Communication\Plugin\Sales                         |
+| ItemMetadataSalesOrderItemCollectionPreDeletePlugin               | Executes before item metadata deletion.                                                                                                                                                                     | None                                                                                                                                             | Spryker\Zed\SalesProductConnector\Communication\Plugin\Sales                         |
+| SalesReclamationSalesOrderItemCollectionPreDeletePlugin           | Executes before reclamation order items deletion.                                                                                                                                                           | None                                                                                                                                             | Spryker\Zed\SalesReclamation\Communication\Plugin\Sales                              |
+| ServicePointSalesOrderItemCollectionPostUpdatePlugin              | Processes service points after order items collection update.                                                                                                                                               | None                                                                                                                                             | Spryker\Zed\SalesServicePoint\Communication\Plugin\Sales                             |
+| ServicePointSalesOrderItemCollectionPreDeletePlugin               | Executes before service point order items deletion.                                                                                                                                                         | None                                                                                                                                             | Spryker\Zed\SalesServicePoint\Communication\Plugin\Sales                             |
```

Comment:
Please adjust the description to be clear what they do (please check all the places in the IG)